class.jetpack-modules-list-table.php

```diff
@@ -76,7 +77,18 @@ function js_templates() {
 			if ( data.items.length ) {
 			_.each( data.items, function( item, key, list ) {
 				if ( item === undefined ) return;
-				#>
+				if ( jetpackModulesData.coreIconAvailable && 'site-icon' == item.module ) { #>
+				<tr class="jetpack-module <# if ( ++i % 2 ) { #> alternate<# } #>" id="site-icon-deprecated">
```

Comment:
Maybe instead of using an id, we could use a more generic class, like 'module-deprecated' so that we can use it for other deprecated modules.